src/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue

```diff
@@ -76,6 +79,13 @@ async function loadFile() {
 
     try {
         await handleFileSource(fileUrl.value, false)
+        if (!fileUrl.value.match(/https:\/\//)) {
+            store.dispatch('addWarnings', {
+                warnings: [
+                    new WarningMessage('import_http_external_file_warning', { url: fileUrl.value }),
```

Comment:
We supply an extra param `url`, but in the translation, we don't use it. Do you think it's necessary?

Reply:
I did the code before the translation, and forgot to remove the unused param. thanks :)